.github/workflows/github-actions-test-installer.yml

```diff
@@ -0,0 +1,46 @@
+name: Test DependencyInstaller script
+on:
+  # Triggers the workflow on push or pull request events
+  push:
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:     
+  testInstaller:
+    strategy:
+      matrix:
+        os: ["ubuntu:20.04", "ubuntu:22.04", "centos:centos7"]
+        args: ["", "-prefix=/home/testUser/openroad-deps"]
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.os }}
+    steps:
+      - name: Setup
+        run: |
+          if [ "${{ matrix.os }}" != "centos:centos7" ]; then 
+            apt-get update
+            apt-get -y install git
+          else
+            yum -y install git
+          fi
+      - name: Checkout repository code with submodules
+        run: |
+          git clone --recursive https://github.com/${{ github.repository }}.git .
+          PR_NUMBER=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')
```

Comment:
Will this only work for PRs? Or will it also get a branch name, e.g., `master`?

Reply:
yes I believe so, I added something more generic in case if it isn't a pull request